<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passcodeMenu</class>
 <widget class="QWidget" name="passcodeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>passcodeMenu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>261</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>71</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Passcode &amp; Finger Print Menu</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_changePasscode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>281</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Change Passcode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_fingerprint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>281</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Finger Print</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>71</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_rmUsrFromMenu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove User from Login Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
